src/System.Drawing.Common/ref/Configurations.props

```diff
@@ -1,10 +1,14 @@
 ﻿<Project>
   <PropertyGroup>
-    <BuildConfigurations>
+    <PackageConfigurations>
       netcoreapp;
       netstandard;
-      netfx;
       net461;
+      uap;
```

Comment:
10.0.16299 or 10.0.16300? 🤔

Reply:
I assume you want to package 16299 and keep building 16300 which is uap.

Reply:
Nope, I want to package latest because we need to typeforward SystemColors down to Primitives which is only available in uap latest (moved in this PR). Uap16299 should just use the netstandard configuration.

Reply:
@ericstj mentioned that we currently don't ship any uap10.0.16300 assets in our packages as we strip them out.

Reply:
Hmmm so then we should just leave the netstandard impl for uap? However since we moved SystemColors down how would we represent that? Should we only include it in netcoreapp for now in Primitives?

Reply:
Talked to @ericstj offline and he said that eventhough we're stripping uap16300 assets from the package, we should still represent it in the configurations for later reference because that type was moved inbox into uap.

Reply:
Perfect! Makes sense.